.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
```

Comment:
We use CirlceCI for our tests/integration and don't plan on moving to github actions right now. 
Could you remove this and add any needed updates to the CircleCI file (including the PHP 8 target)? Ive enabled circleCI to run on PR branches just now - so hopefully it runs next time you push an update.

Reply:
For now I've added to PHP8 compatibility branch. If the build fails, I'm going to take circleci out of this, as it is out of scope for me. I know they deprecated some of the images the way they are being called, and it's not really part of this work.